app/src/main/java/com/twilio/conversations/app/common/extensions/TwilioExtensions.kt

```diff
@@ -309,7 +314,9 @@ suspend fun User.setFriendlyName(friendlyName: String): Unit = suspendCoroutine
 }
 
 suspend fun Message.getMediaContentTemporaryUrl(): String = suspendCoroutine { continuation ->
```

Comment:
Just drop this method and introduce `Media.getTemporaryContentUrl`. So taking into account my suggested changes above you can call it like:
```
val url = message.firstMedia?.getTemporaryContentUrl()
```

Reply:
Fixed